.github/workflows/i18n-consistency-checker.yaml

```diff
@@ -79,6 +80,7 @@ jobs:
             fi
           done
       - name: Create Issue
+        if: ${{ <expression> }}
```

Comment:
This is not working as is.

However i was wondering if here we could test for the existence of `issue.md` and only run this entire step if that file exists?

Reply:
That implementation should be in place below, but it doesn't seem to be working.
If there were a similar issue, it would update the new edits as a comment to that issue, so that a new issue would not be created.

Also, since the script is doing the checking for the issue's existence here, there should be an Action in the previous section to make that determination. The process would be like splitting the existing code.

```
            if expr "$existing_issue_id" : '^[0-9]*$' >/dev/null; then
              gh issue comment "$existing_issue_id" -F issue.md -R $GITHUB_REPOSITORY
            else
              gh issue create -t "$issue_title" -F issue.md -R $GITHUB_REPOSITORY -l documentation
            fi
```